rapid7_insightidr/plugin.spec.yaml

```
@@ -564,27 +564,27 @@ actions:
         type: logsets_info
         required: true
 
-  advanced_query:
-    title: Advanced Query
-    description: Realtime query into Insight IDR logs
+  advanced_query_on_log:
+    title: Advanced Query on Log
+    description: Realtime query into an Insight IDR log. This will query individual logs for results
     input:
       query:
         title: Query
-        description: Query
+        description: LQL Query
         type: string
         example: "where(user=adagentadmin, loose)"
         required: true
       time_from:
         title: Time From
-        description: Beginning time and date for the query
+        description: Beginning time and date for the query. The format is flexible and will work with simple dates to full ISO time
```

We should be showing examples of both on all these descriptions

```suggestion
        description: Beginning time and date for the query. The format is flexible and will work with simple dates (e.g. 01-01-2020) to full ISO time (e.g. 01-01-2020T00:00:00)
```